docs/design-proposals/node/Unified-Vertical-Scaling-for-Container-and-VM-Pods.md

```diff
@@ -55,17 +58,23 @@ some, like VMs are more sensitive, especially when using low number of Pod repli
 Moreover, for stateful VMs or batch workloads, Pod restart is a serious disruption,
 resulting in lower availability or higher cost of running.
 
-Allowing Resources to be changed without recreating the Pod or restarting the
-Containers or VMs addresses this issue directly.
+Allowing resources to be changed without recreating the Pod or restarting the
+containers or VMs addresses this issue directly.
 
 ### Goals
 
 * Primary: allow to change Pod resource requests & limits without restarting
   its Containers or VMs.
 * Secondary: allow actors (users, VPA, StatefulSet, JobController) to decide
   how to proceed if in-place resource resize is not possible.
+* For VM, NUMA awareness of memory increase 
+* For VM, x86 CPU architecture must be supported, ARM can be stretch goal
+* For VM, major vendors of linux guest OS,
 
 ### Non-Goals
+* For VM, windows guest nor windows host to be supported
```

Comment:
consiner support VM

Reply:
removed from no-goal. windows VM should be part of it as implementation is OS neutral